### DISCUSSION TITLE
Delete INVOICE_EXTENDED_WORKFLOW and Finance 2 role?

### DISCUSSION THREAD

**Discussion by frjo** [OpenTechFund/opentech.fund#4121] [General]
OTF is not planing to use the INVOICE_EXTENDED_WORKFLOW, can we delete all related code completely?

Is the Finance 2 "role" needed? If not we can remove that as well.

**Comment by frjo** [comment]
@wes-otf  I made a test PR here https://github.com/HyphaApp/hypha/pull/4122

If we do not need Finance 2 then this PR removes a lot of code and complexity.

**Comment by wes-otf** [comment]
this slipped by me last week but I don't believe we OTF needs it. Will confirm today though so we can get this through.
